docs/_sass/_base.scss

```diff
@@ -70,3 +70,37 @@ table:not(.table) {
         background-color: #f9f9f9;
     }
 }
+
+figure {
+  text-align: center;
+
+  img {
+    border: 1px solid #e5e5e5;
+    margin: 0 auto;
+    padding: .5rem;
+  }
+
+  figcaption {
+    color: #999;
+    font-size: .8rem;
+    margin-top: 1rem;
+  }
+}
+
+.callout {
+  border-bottom-right-radius: 50px;
+  border-top-right-radius: 50px;
+  margin-left: -50px;
+  padding: 1rem 0 1rem 50px;
```

Comment:
If you started using rem in padding why not in margin too?

Reply:
Fixed left & right margins, relative vertical rhythm.